HelpfulItem.h

```diff
@@ -10,6 +10,6 @@
 struct HelpfulItem : Item
 {
     HelpfulItem() : Item("helpful item", 2) { }
-    // void use(Character* character) override;
+    void use(Character* character) override;
 };
```

Comment:
Don't **include** what you don't use!  A bunch of files are being included above that aren't being used
- [ ] revise